streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1256,9 +1137,9 @@ public void apply(final Task task) {
         };
 
         sb.append(indent).append("\tActive tasks:\n");
-        taskManager.performOnActiveTasks(printAction);
+        taskManager.performOnActiveTasks(printAction, false);
```

Comment:
We've only ever printed the active tasks. Have a look a couple of revisions ago:
```
 if (activeTasks != null) {
            sb.append(indent).append("\tActive tasks:\n");
            for (final Map.Entry<TaskId, StreamTask> entry : activeTasks.entrySet()) {
                final StreamTask task = entry.getValue();
                sb.append(indent).append(task.toString(indent + "\t\t"));
            }
        }

        // iterate and print standby tasks
        if (standbyTasks != null) {
            sb.append(indent).append("\tStandby tasks:\n");
            for (final StandbyTask task : standbyTasks.values()) {
                sb.append(indent).append(task.toString(indent + "\t\t"));
            }
            sb.append("\n");
        }
```
However, i see no reason why we shouldn't print all of them. So i'll put a print on `TaskManager` -> `AssignedTasks`